src/backend/CommandBufferStateTracker.cpp

```diff
@@ -289,9 +290,12 @@ namespace backend {
             builder->HandleError("Can't use a compute pipeline while a render pass is active");
             return false;
         }
-        aspects.set(VALIDATION_ASPECT_COMPUTE_PIPELINE);
+        if (!SetPipelineCommon(pipeline)) {
```

Comment:
Now we could revert this to ```return SetPipelineCommon(pipeline)``` like it did before, as the ordering doesn't matter anymore. (sorry for all the back and forths.)